Update page design

Nikolay-Gavazov opened the pull request:
Made the colors and positions on the components. Add date and show more button.

Nikolay-Gavazov committed:
Update page design

Made the colors and positions on the components. Add date and show more button.

Nikolay-Gavazov committed:
Update events.page.html

Getsov committed:
Merge branch 'dev' into NG-All-Events

Nikolay-Gavazov committed:
Change the icons.

Replaced the icons with the ones from the design.

Nikolay-Gavazov commented:
Replaced the icons with the ones from the design.

Getsov commented on src/app/tabs/tabs.page.html:
```diff
@@ -2,7 +2,10 @@
     <ion-tab-button tab="home" href="/home">
-      <ion-icon aria-hidden="true" name="home"></ion-icon>
+      <!-- <ion-icon aria-hidden="true" name="home"></ion-icon> -->
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```
Is this svg working when tab is active?

Nikolay-Gavazov replied:
![Screenshot 2023-11-13 111249](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/2674e573-193c-49cd-a151-ea65b0c3ebc2)

Getsov replied:
So it’s not working, right? See the design

Getsov commented on src/app/tabs/tabs.page.html:
```diff
@@ -12,17 +15,20 @@
     <ion-tab-button tab="profile" href="/profile">
-      <ion-icon aria-hidden="true" name="person"></ion-icon>
+      <!-- <ion-icon aria-hidden="true" name="person"></ion-icon> -->
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```
Same here - is this working when highlighting active tab?

Nikolay-Gavazov replied:
![Screenshot 2023-11-13 111321](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/017d4b00-b461-4097-aaa9-f0c4b25fd9ac)

Getsov replied:
The same here

Nikolay-Gavazov committed:
Fixed the icons

I had to add the SVG as a source and delete its default colors and add a class, after an hour of digging, I found it.

Nikolay-Gavazov commented:
![Screenshot 2023-11-13 12](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/0fe2b2a0-e844-4dc3-965b-d7deaf217cd1)
![Screenshot 2023-11-13 111321](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/450445c0-7a31-4339-8728-af57b80ac41e)

Done

Nikolay-Gavazov commented:
![Screenshot 2023-11-13 111](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/393b50c3-e9b6-4bf5-9e44-2f13818f5914)
![Screenshot 2023-11-13 111249](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/a08c0691-6190-41d7-a004-c50c1330bb6d)

Done
